scripts/rocket-chat-pr.js

```diff
@@ -0,0 +1,93 @@
+//
+//
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Jason Leach on 2019-03-26.
+//
+
+/* eslint-disable camelcase, class-methods-use-this */
+
+const labelsAsString = labels => labels.map(label => label.name).join(', ');
+
+const githubEventHandlers = {
+  //
+  // Pull Request Trigger
+  //
+
+  pull_request(request) {
+    let message;
+    if (
```

Comment:
how about a case switch here?

Reply:
I think you generally want to use a `switch` when comparing the non-trivial value of an expression while an `if` is better if you're just checking the truthiness of an expression. For example, if I was testing the value of `request.content.action` this I would use a switch:
```
      request.content.action === 'opened' ||
      request.content.action === 'reopened' ||
      request.content.action === 'edited' ||
      request.content.action === 'synchronize'
```

But when checking the truthiness`(request.content.action === 'reopened' || request.content.action === 'edited')` then I think a switch is better.